### PR TITLE
silly bug read grid

### PR DESCRIPTION
**Pull request by raphaeldussin**
sorry I missed that! that was a stupid bug that affected only padded grids.

### CODE REVIEW

**Comment by rabernat** [comment]
A good question to ask is: why was this not covered by our test suite? What test could we add to prevent such a bug from ever entering the code base?

File: xmitgcm/test/test_utils.py [hunk 1]
```diff
@@ -1,5 +1,14 @@
-from xmitgcm.test.test_xmitgcm_common import *
+import pytest
+import os
+import numpy as np
 import xarray
+import dask
+from xmitgcm.test.test_xmitgcm_common import hide_file
+from xmitgcm.test.test_xmitgcm_common import all_mds_datadirs
+from xmitgcm.test.test_xmitgcm_common import mds_datadirs_with_diagnostics
```
[the remark this answers is not in the document]
Comment (raphaeldussin) [reply xmitgcm/test/test_utils.py]: this fixture needs to be imported for pytest to work

File: xmitgcm/test/test_utils.py [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+from xmitgcm.test.test_xmitgcm_common import llc_mds_datadirs
+from xmitgcm.test.test_xmitgcm_common import all_grid_datadirs
```
[the remark this answers is not in the document]
Comment (raphaeldussin) [reply xmitgcm/test/test_utils.py]: this fixture needs to be imported for pytest to work

File: xmitgcm/test/test_utils.py [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+from xmitgcm.test.test_xmitgcm_common import _experiments
```
[the remark this answers is not in the document]
Comment (raphaeldussin) [reply xmitgcm/test/test_utils.py]: this fixture needs to be imported for pytest to work

**Pull request merged**